explorer/client/src/utils/utility_functions.tsx

```diff
@@ -0,0 +1,59 @@
+import mockTransactionData from './mock_data.json';
```

Comment:
Similar to Sam's [comment](https://github.com/MystenLabs/sui/pull/454#discussion_r807128575), let's avoid having a general util file and create `navigation_utils`, `bytes_conversion_utils`, `logging_util`.. instead, or move to the file in which they are used if there's only one

Reply:
ok, i'll  see what organization makes sense and move things around

Reply:
reorganized into 3 files:
`searchUtil.ts` : search function
`mockUtil.ts`: mock data function
`stringUtils.ts` : string manipulation & testing functions